File: src/main/java/io/github/chindeaone/collectiontracker/collections/GemstonesManager.java
package io.github.chindeaone.collectiontracker.collections;


public class GemstonesManager {

    public static String[] gemstones;

    public static boolean checkIfGemstone(String collectionName) {
        for(String gemstone : gemstones) {
            if (collectionName.equalsIgnoreCase(gemstone)) {
                return true;
            }
        }
        return false;
    }
}

File: src/main/java/io/github/chindeaone/collectiontracker/collections/GemstonesManager.kt
package io.github.chindeaone.collectiontracker.collections


object GemstonesManager {
    var gemstones: MutableList<String> = mutableListOf()

    fun checkIfGemstone(collectionName: String): Boolean {
        for (gemstone in gemstones) {
            if (collectionName.equals(gemstone, ignoreCase = true)) {
                return true
            }
        }
        return false
    }
}